src/org/thoughtcrime/securesms/ShareActivity.java

```diff
@@ -201,6 +201,17 @@ public void onMediaResolved(Uri uri) {
 
   private void handleResolvedMedia(Intent intent) {
     int       chatId           = intent.getIntExtra(EXTRA_CHAT_ID, -1);
+
+    String[] extraEmail = getIntent().getStringArrayExtra(Intent.EXTRA_EMAIL);
```

Comment:
maybe add a comment or a link about how the intent is typically fired, which apps support that etc.

Reply:
pretty long, but complete :)

Reply:
BTW in case thats not clear: The other app does not have to be specifically adapted for this. I tried with SuperFreezZ (Settings -> About -> Send logs -> Select Delta Chat) and it worked out of the box.

https://gitlab.com/SuperFreezZ/SuperFreezZ/-/blob/master/src/superfreeze/tool/android/userInterface/settingsActivity/SettingsActivity.kt#L253
```
				val sharingIntent = Intent(Intent.ACTION_SEND)
				sharingIntent.type = "text/plain"
				sharingIntent.putExtra(Intent.EXTRA_EMAIL, arrayOf("superfreezz-automated@gmx.de"))
				sharingIntent.putExtra(Intent.EXTRA_TEXT, "Text you want to share here")
				startActivity(Intent.createChooser(sharingIntent, getString(R.string.share_logs)))
```
It's shorter because it only sends text.

Reply:
sure! my initial concern was sending files failed with DC, while I was successful with gmail and bluemail apps.... not a media (pics, video), but some binary data.... so, I opened this PR.